src/MyQConnector/Connectors/MyQConnector.cs

```diff
@@ -0,0 +1,199 @@
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Serilog;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Linq;
+
+namespace Innovatrics.SmartFace.Integrations.MyQConnectorNamespace.Connectors
```

Comment:
MyQConnectorNamespace je trochu creepy, pouzivat "Namespace" v namespace pripadne "Class" v nasve triedy je anti-pattern. Doporucil by som to zmenit na MyQConnector a potom upravit nazov triedy (a interface) na Connector a IConnector

Reply:
Upravene, fixnute